addons/DataBindControls/DataBindingsGlobal.gd

```diff
@@ -9,14 +9,14 @@ var _change_detection_queued := false
 
 
 ## queue change detection
-func detect_changes() -> void:
+func detect_changes(ancestor: Node = null) -> void:
 	if _change_detection_queued:
 		return
 	_change_detection_queued = true
-	call_deferred("_detect_changes")
+	call_deferred("_detect_changes", ancestor)
 
 
-func _detect_changes():
+func _detect_changes(ancestor: Node):
 	# TODO: queue change detection per viewport root or control root?
```

Comment:
This implements change detection per control root, that part of the TODO should be removed